src/components/nodes/pool/pool.vue

```diff
@@ -101,6 +101,15 @@ export default {
 
       return this.graph.findModelsInArea(area);
     },
+    moveEmbeddedElements(currentElement, toPool) {
+      this.getElementsUnderArea(currentElement).forEach((element) => {
+        if (element.isEmbeddedIn(currentElement)) {
+          const elementDefinition = element.component.node.definition;
+          pull(this.containingProcess.get('flowElements'), elementDefinition);
+          toPool.component.containingProcess.get('flowElements').push(elementDefinition);
+        }
+      });
```

Comment:
This could be re-written to be a little clearer:
```suggestion
     this.getElementsUnderArea(currentElement)
     .filter(element => element.isEmbeddedIn(currentElement))
     .map(element => element.component.node.definition)
     .forEach((elementDefinition) => {
           pull(this.containingProcess.get('flowElements'), elementDefinition);
           toPool.component.containingProcess.get('flowElements').push(elementDefinition);
       });
```